tests/test_validator.py

```diff
@@ -0,0 +1,29 @@
+from copy import deepcopy
+
+from lark import Lark
+
+from omnidoc.tree_validator import validate_tree
+
+p = Lark("""
```

Comment:
I thought we wanted to keep the ontology/schema in grammar and the actual tree in the text? That makes things easier, no? Sorry and correct me if I have wrong understading.

Reply:
Not sure what you're asking... The ontology is in the grammar, and you can produce trees according to the ontology. What's "the text"?

Reply:
I'll think and get back to you with more concrete answer.